application/__init__.py

```diff
@@ -17,13 +17,21 @@
 #   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 #   FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
 #   IN THE SOFTWARE.
-
+import logging.config
+import yaml
 import os
 import sys
 
 from flask import Flask, Blueprint
 from werkzeug.serving import WSGIRequestHandler
 
+with open('logging_default_config.yaml', 'rt') as config_file:
```

Comment:
```
Traceback (most recent call last):
  File "R:\iCtrl\ictrl_be.py", line 22, in <module>
    from application import api, app, APP_HOST, APP_PORT
  File "R:\iCtrl\application\__init__.py", line 28, in <module>
    with open('logging_default_config.yaml', 'rt') as config_file:
FileNotFoundError: [Errno 2] No such file or directory: 'logging_default_config.yaml'
```

When running ictrl_be.py the above error shows. Let's move this file to the project root.

Reply:
Why do we need `t` in the mode?

Reply:
Encode as text, I guess that is default already so we don't need it